### PR TITLE
created an interface for the standard Channel ports using #2255 as an…

### PR DESCRIPTION
**Pull request by menaman123**
… example interface change set

| | |
|:---|:---|
|** #2585**|  |
|**No**|  |
|**No**|  |

---
## Change Description

Create Telemetry/Channel Interface #2585

## Rationale

Part of Create F´ Interfaces From Existing Concepts #2578

## Testing/Review Recommendations

None

## Future Work

Continue #2578

### CODE REVIEW

**Comment by LeStarch** [comment]
The file is empty.  Can you fill it in?

File: Fw/Interfaces/ChannelInterfaces.fppi [hunk 1]
```diff
@@ -0,0 +1,2 @@
+@ Telemetry port
+telemetry port tlmOut
```
Comment (LeStarch) [inline Fw/Interfaces/ChannelInterfaces.fppi:2]: Last step is to delete line 68-69 in `SignalGen.fpp` as your interface adds that port!

File: Ref/SignalGen/SignalGen.fpp [hunk 2]
```diff
@@ -93,6 +93,12 @@ module Ref {
 
     include "Events.fppi"
 
+    # ----------------------------------------------------------------------
+    # Interfaces
+    # ----------------------------------------------------------------------
+
+    include "../../Fw\Interfaces\ChannelInterfaces.fppi"
```
Comment (LeStarch) [inline Ref/SignalGen/SignalGen.fpp]: You should use `/` instead of `\` as well.

Comment (thomas-bc) [reply Ref/SignalGen/SignalGen.fpp]: Should the channel interface be used for all components in Ref and RPI as well? Or are we happy to keep it hybrid?

Comment (LeStarch) [reply Ref/SignalGen/SignalGen.fpp]: Not just yet.  Let's get it in and proven in SignalGen, then we can slowly refactor.

**Review by LeStarch — APPROVED** [review]

**Comment by thomas-bc** [comment]
Closing in favor of https://github.com/nasa/fprime/pull/2757/files

**Pull request closed without merging**